LibreNMS/IRCBot.php

```diff
@@ -569,7 +569,13 @@ private function _help($params)
 
     private function _version($params)
     {
-        return $this->respond($this->config['project_name_version'].', PHP: '.PHP_VERSION);
+        $versions       = version_info(false);
+        $schema_version = $versions['db_schema'];
+        $version        = `git rev-parse --short HEAD`;
```

Comment:
We already have: local_sha

You'll just need to cut it down to a shorter version. Better than doing another exec call.